my-sms/spec/cypress/integration/acceptance/display_student_list.spec.js

```diff
@@ -0,0 +1,106 @@
+context("Test students list page content", () => {
+  describe("For a single student", () => {
+    beforeEach(() => {
+      cy.app("clean");
+      cy.appFactories([
+        [
+          "create",
+          "student",
+          {
+            id: 1,
+            email: "john@example.com",
+            title: null,
+            first_name: "John",
+            last_name: "Smith",
+            middle_name: null,
+            birth_date: "2011/04/09",
+            gender: "female",
+          },
+        ],
+      ]);
+      cy.visit("/students");
+    });
+
+    it("Shows the students page", () => {});
+
+    it("Shows a table with id, full name, email, birth date, gender, and created at columns", () => {
+      const cols = [
+        "Id",
+        "Full name",
+        "Email",
+        "Birth date",
+        "Gender",
+        "Created at",
+      ];
+      cy.get("thead > tr").within(() => {
+        cols.forEach((e, i) => cy.get("th").eq(i).contains(e));
+      });
+    });
+
+    it("Shows a student with the correct id, full name, email birth date, and gender attributes", () => {
+      const attrs = [
+        1,
+        "John Smith",
+        "john@example.com",
+        "2011-04-09",
+        "female",
+      ];
+      cy.get("tbody > tr").within(() => {
+        attrs.forEach((e, i) => cy.get("td").eq(i).contains(e));
+      });
+    });
+
+    it("Shows a student with a created at attribute in the correct format", () => {
+      cy.get("tbody > tr > td")
+        .eq(5)
+        .contains(/\d{4}-\d{2}-\d{2}\s\d{2}\:\d{2}\:\d{2}\s\w{3}/);
+    });
```

Comment:
- You want to be very careful about tying you tests to the markup, ie you dont want to have to update the test just because the markup changes but the content is the same. So I suggest you just assert that the Person's details are visible.
Also worth noting that these cypress tests will be much slower than unit tests, so I don't suggest doing a separate `it` for each assertion.

example
```suggestion
    it("lists the student", () => {
      // Shows a table with id, full name, email, birth date, gender, and created at columns
      cy.contains("Id").should("be.visible");
      cy.contains("Full name").should("be.visible");
      cy.contains("Email").should("be.visible");
      cy.contains("Birth date").should("be.visible");
      cy.contains("Gender").should("be.visible");
      cy.contains("Created at").should("be.visible");

      // it("Shows a student with the correct id, full name, email birth date, and gender attributes", () => {
      cy.contains("1").should("be.visible");
      cy.contains("John Smith").should("be.visible");
      cy.contains("john@example.com").should("be.visible");
      cy.contains("2011-04-09").should("be.visible");
      cy.contains("female").should("be.visible");
    });
```